spec/src/main/asciidoc/core/injectionandresolution.asciidoc

```diff
@@ -649,6 +649,9 @@ PaymentProcessor pp = anyPaymentProcessor.select(qualifier).get().process(paymen
 
 In this example, the returned bean has qualifier `@Synchronous` or `@Asynchronous` depending upon the value of `synchronously`.
 
+Qualifiers specified at `Instance` injection point or via via the `Instance.select()` method are used to perform typesafe resolution.
+If no qualifier is specified at `Instance` injection point or via the `Instance.select()` method the `@Default` qualifier is assumed.
+
```

Comment:
I find these sentences redundant.

The section "5.6.1. The Instance interface" clearly states that:
> For an injected Instance, the required qualifiers are the qualifiers specified at the injection point.

And "3.8. The default qualifier at injection points" states:
> If an injection point declares no qualifier, the injection point has exactly one qualifier, the default qualifier @Default.

I understand that section "5.6. Programmatic lookup" is not normative, just a general description and examples. Still, I would reword the sentences to reference the above mentioned sections or to be more PaymentProcessor-example-related.